src/Paket.Core/Dependencies/NuGetCache.fs

```diff
@@ -331,6 +331,21 @@ let GetTargetUserNupkg (packageName:PackageName) (version:SemVerInfo) =
 let GetTargetUserToolsFolder (packageName:PackageName) (version:SemVerInfo) =
     DirectoryInfo(Path.Combine(Constants.UserNuGetPackagesFolder,".tools",packageName.CompareString,version.Normalize())).FullName
 
+let TryGetFallbackFolder () =
+    let dotnet = if isUnix then "dotnet" else "dotnet.exe"
```

Comment:
can we do it without calling dotnet?

Reply:
I'm just using it to find the folder, it's not executed. I guess the alternative is to try the default folder depending on platform.

Reply:
ah I see. that's neat